app/code/Magento/Checkout/Controller/Sidebar/UpdateItemQty.php

```diff
@@ -66,7 +66,7 @@ public function __construct(
     public function execute()
     {
         $itemId = (int)$this->getRequest()->getParam('item_id');
-        $itemQty = $this->getRequest()->getParam('item_qty') * 1;
+        $itemQty = (int)$this->getRequest()->getParam('item_qty') * 1;
```

Comment:
I'm pretty sure qty might be float
```suggestion
        $itemQty = (float)$this->getRequest()->getParam('item_qty') * 1;
```

Reply:
Totally agree with @ihor-sviziev

Reply:
Thanks for advice guys!